test/cli/statsd-invalid-link/test.out

```diff
@@ -0,0 +1,12 @@
+No errors! Great job.
+Name or service not known
+Exception::raise(): statsd initialization failed: host: f4K#l1N&.com port: 8200 prefix: foo.bar.counters
+
+Backtrace:
+  #3 0xef93fc sorbet::StatsD::submitCounters()
```

Comment:
These are memory addresses. You cannot put these in exp files, as they change every time we rebuild sorbet

Reply:
Use sed and/or grep to capture the part of the output that you want

Reply:
fixed!